src/shaping/shaping_linux.go

```diff
@@ -301,7 +332,7 @@ func setupRootQdisc(link_name string) error {
 	}
 	for _, q := range qdiscs {
 		if err := netlink.QdiscDel(q); err != nil {
-			return fmt.Errorf("Could not delete qdisc: %v", err)
+			Log.Println("warning: Could not delete root qdisc:", err)
```

Comment:
Trying to delete the root qdisc on `veth` interfaces fails. Creating a new root qdisc over top of the existing one produces the desired effects.

Once [netlink#64](https://github.com/vishvananda/netlink/pull/64) lands, we can change this logic to a replace call instead of delete/create.